app/controllers/solidus_bolt/base_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'base64'
```

Comment:
This `require` is duplicated in more than one place.
You can require it in one place from [lib/solidus_bolt.rb](https://github.com/nebulab/solidus_bolt/blob/master/lib/solidus_bolt.rb#L3-L9), you can also consider removing the `require` from the others controllers.

Reply:
after update it will be only in this controller, so I suppose I can leave it there

Reply:
see https://github.com/nebulab/solidus_bolt/commit/cbad318944309509197f3931924794104865de47